### PR TITLE
Remove Python API ref page, Spack Install, fix some links

### PR DESCRIPTION
**Pull request by brenthuisman**

### CODE REVIEW

**Comment by brenthuisman** [comment]
Build fails because one job ran out of time. I see no reason for actual breakage.

File: doc/gs_single_cell.rst [hunk 1]
```diff
@@ -57,15 +56,15 @@ Let's unpack that.
...
+segment tree which is a list of segments, which are tapered cones with a *tag*.
+:meth:`arbor.segment_tree.append` takes 4 arguments, starting with
 the parent segment. The first segment added has no parent however, which is made clear by
 using :class:`arbor.mnpos`. Then two :class:`arbor.mpoint` s are supplied, the proximal
 and distal endpoints of the segment. Finally, an integer value can be supplied to tag the
 segment for future reference.
 
-In step **(2)** a dictionary of labels is created using :class:`arbor.label_dict<arbor.
-_arbor.label_dict>`. Cell builders need to refer to *regions* and *locations* on a cell
+In step **(2)** a dictionary of labels is created using :class:`arbor.label_dict<arbor._arbor.label_dict>`.
```
Comment (noraabiakar) [inline doc/gs_single_cell.rst:66]: do we need `_arbor` here?

Comment (noraabiakar) [reply doc/gs_single_cell.rst:66]: It's also present in `doc/conf.py`

**Review by noraabiakar — APPROVED** [review]
lgtm

**Pull request merged**